src/sample/infrastructure/repository/testdata/index.ts

```diff
@@ -0,0 +1,17 @@
+import { plainToClass } from 'class-transformer/build/package';
+import { SampleAggregate } from '../../../domain/aggregate/Sample';
+
+const singleSampleData: SampleAggregate = plainToClass(SampleAggregate, {
+  id: 1,
+});
+
+const multipleSampleData: SampleAggregate[] = plainToClass(SampleAggregate, [
```

Comment:
```suggestion
const sampleData: SampleAggregate[] = plainToClass(SampleAggregate, [
```
だけでいい、`singleSampleData `が要らん。